File: Daily Coding/c/alternate_sort.c
// Alternate sorting (first, last, second, second last...)
#include <stdio.h>
int main()
{
    int a[50], b[50], i, j, len;
    printf("Enter the length of the array: ");
    scanf("%d",&len);
    printf("Enter the array: ");
    for(i=0; i<len; i++)
        scanf("%d",&a[i]);
    j=0;
    i--;
    while(j < i)
    {
        printf("%d ",a[i]);
        printf("%d ",a[j]);
        j++;
        i--;
        if(i==j)
            printf("%d ",a[i]);
    }
    return 0;
}

File: Daily Coding/python/alternate_sort.py
# Alternate sorting (first, last, second, second last...)
a = []
len = int(input("Enter the number of elements: "))
print("Enter the array: ")
for i in range(len):
    temp = int(input())
    a.append(temp)
i = 0
len -= 1
while i < len:
    print(a[len],end=" ")
    print(a[i],end=" ")
    i += 1
    len -= 1
    if i == len:
        print(a[i])
        break
